### ISSUE TITLE
Model import from StochML broken

### ISSUE THREAD

**Issue by ahellander** [JohnAbel/gillespy#21]
The code for importing a model form a StochKit XML exported from StochSS  is broken due to changes in the Reaction constructor.

**Comment by JohnAbel** [comment]
Is this issue still true?

**Comment by ahellander** [comment]
Yes, but I have a fix for it in a branch. Will send a PR next week when back from vacation. 

Mvh
Andreas

————————————————————————
Andreas Hellander
Associate Professor in Scientific Computing
Project Leader SNIC Science Cloud
Department of Information Technology, Uppsala University
andreas.hellander@it.uu.se

**Comment by JohnAbel** [comment]
PR please!
